pdata/ptrace/alias.go

```diff
@@ -0,0 +1,62 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ptrace // import "go.opentelemetry.io/collector/pdata/ptrace"
+
+import "go.opentelemetry.io/collector/pdata/internal"
+
+// This file contains aliases for trace data structures.
+
+// TracesMarshaler is an alias for internal.TracesMarshaler interface.
+type TracesMarshaler = internal.TracesMarshaler
+
+// TracesUnmarshaler is an alias for internal.TracesUnmarshaler interface.
+type TracesUnmarshaler = internal.TracesUnmarshaler
+
+// TracesSizer is an alias for internal.TracesSizer interface.
+type TracesSizer = internal.TracesSizer
```

Comment:
Question: Should we remove for these the prefix "Traces"?

Reply:
Yes, I think this prefix isn't needed given that we still have `TracesSize` method in the interface